src/components/settings/organization/general.tsx

```
@@ -0,0 +1,254 @@
+    return
+
+    setIsLoading(true)
+
+    const { error } = await authClient.organization.delete({
+      organizationId: organization.id
+    })
+
+    setIsLoading(false)
+    if (!error) {
+      toast.success("Organization successfully deleted!")
+    }
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)